lib/Controller/FederatedInvitesController.php

```diff
@@ -239,9 +239,19 @@ public function inviteAccepted(string $token = '', string $provider = ''): JSONR
 			$recipientProvider = $this->federatedInvitesService->getProviderFQDN();
 			$client = $this->httpClient->newClient();
 			$responseData = null;
+			$provider_url = parse_url($provider);
```

Comment:
`parse_url` does not work in this case, without a scheme. Eg. when the OCM address is `sometoken@nc-ocm.nl`
Then parse url results in `Array ( [path] => nc-ocm.nl ) `

parse_url needs a scheme.